roles/sap_hypervisor_node_preconfigure/tasks/platform/redhat_ocp_virt/create-sap-bridge.yml

```diff
@@ -1,6 +1,10 @@
 ---
 - name: Create SAP bridge NodeNetworkConfigurationPolicy
   kubernetes.core.k8s:
+    host: "{{ sap_hypervisor_node_preconfigure_ocp_endpoint }}"
```

Comment:
Might be easier to export the auth settings in env variables, to avoid adding them to every single invocation of a module?

Reply:
Yes, agreed. I've solved it via delegate_to and setting the environment there but you seem not to be too happy with that (comment below).